libs/platform/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Just like my other comment, when we go 9, we want to also with Ivy, unless we have some specific reason .

Reply:
`IVY` is not ready to be used on library level I think

Reply:
You are right. To publish Ivy compiled code does not make sense at this time.